var/spack/repos/builtin/packages/suite-sparse/package.py

```diff
@@ -99,6 +99,12 @@ def install(self, spec, prefix):
             'LAPACK=%s' % spec['lapack'].libs.ld_flags,
         ]
 
+        # 64bit blas in UMFPACK:
+        if (spec.satisfies('^openblas+ilp64') or
+            spec.satisfies('^intel-mkl+ilp64') or
+            spec.satisfies('^intel-parallel-studio+mkl+ilp64')):
+            make_args.append('UMFPACK_CONFIG=-DLONGBLAS=\'long long\'')
```

Comment:
Can you use double quotes for the string? I have a personal vendetta against escaped quotes when they aren't necessary.

Reply:
@adamjstewart done.